.github/workflows/ci.yml

```diff
@@ -26,10 +26,14 @@ jobs:
           - '3.0'
           - '3.1'
           - '3.2'
+          - '3.3'
```

Comment:
I'm gonna steal this line right away and it should disappear after a rebase... leave the inclusion below tho

Reply:
Rebased.